### PR TITLE
feat: support for transparent colorschemes

### PR DESCRIPTION
**Pull request by loqusion**
Since `guibg` isn't set for `Normal` in transparent colorschemes, the user needs to manually supply a value (steps for doing so are documented). Rather than having to specify an entire `colors` table, the user may use a single `bg` value for convenience.

### CODE REVIEW

**Comment by loqusion** [comment]
(also not sure why the transparency looks so bad in the gif, it looks completely different on my screen)

**Comment by HampusHauffman** [comment]
Looks great! i will test this hopefully tomorrow or sunday and then merge it! Thank you!

**Comment by mightyiam** [comment]
How can I find out whether this lack of support (by default) is the cause of no blocks, please?

**Comment by loqusion** [comment]
> How can I find out whether this lack of support (by default) is the cause of no blocks, please?

Apologies, not quite sure what you mean 😅 could you rephrase?

**Comment by mightyiam** [comment]
I'm pretty sure I have a transparent background. I'm just going to wait for this to be merged and try it.

**Review by HampusHauffman — APPROVED** [review]

**Comment by HampusHauffman** [comment]
Thank you  @loqusion !
